Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine
+
+RUN mkdir /core
+COPY src/ /core/
```

Comment:
I think the logs directory (as all the others) should be created here and not inside the code.

Reply:
I guess I could do that, I did it how I did because its faster for me to debug the code natively on my machine and then test in the docker image